index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: setup & mobile first</title>
+    <!-- link to style.css -->
+    <link rel="stylesheet" href="style.css">
+    <!-- link to roboto & poppins & RoundFamily & Inter fonts: copied from google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins:wght@300&family=Roboto&display=swap" rel="stylesheet">
+    <!-- link to my burger icon -->
+    <script src="https://kit.fontawesome.com/8f4917926e.js" crossorigin="anonymous"></script>
+</head>
+
+
+<body>            
+    <!-- my header with a navegation menu. Contains my logo and my "burger" menu button -->
+    <header class="header">
+        <nav class="nav">
+        <div class="my-logo"><h1 >My logo</h1></div>    
+            <!-- import my burger menu from Font Awsome-->
+        <div class="burger-menu"><i class="fa-solid fa-bars"></i></div>    
+        </nav>
+
+         <!-- main content on my web -->  
+        <main class="main">
+            <div>
+            <h2 class="h-1">Hi there.<br>I'm Nico.</h2>
+            <h3 class="h-2">I'm a Sofware Developer</h3>
+            <p class="para-1">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+            </div>
+            <!-- social medias into <div> -->
+            <div class="social-medias">
+                <a href="" class="githubcat"><img src="GitHubcat.png" alt="GitHubcat"></a>
+                <a href="" class="linkedin"><img src="Linkedin.png" alt="Linkedin"></a>
+                <a href="" class="angellist"><img src="Angellist.png" alt="Angellist"></a>
+                <a href="" class="twitter"><img src="Twitter.png" alt="Twitter"></a>
+                <a href="" class="medium"><img src="Medium.png" alt="Medium"></a>    
+            </div>
```

Comment:
- [ ] Excellent job building your social media list 🚀. However, please kindly make sure that you use the semantic `ul` and `li` tags to build your lists instead of a `div` tag and the `a` tags alone 🤓.
Here is a small example of how a list with semantic HTML tags looks 👇🏽

```
<ul>
  <li><a href="#">First item</a></li>
  <li><a href="#">Second item</a></li>
  <li><a href="#">Third item</a></li>
</ul>
```